docker/local-rest-scorer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
```

Comment:
Let's consider having a multi-stage image build as done here: https://github.com/h2oai/h2oai-storage/blob/master/Dockerfile.
This should allow use to have a slim resulting image (no ubuntu, no jdk) without manually cleaning stuff.
There should be some jdk8 images for the build phase as well as some slim jre only images for the runtime image (some time ago, alpine was popular, not sure about the preferred choice now).

Reply:
Moving to `8-jre-alpine` takes us from 472MB to 117MB, including the scoring runtime (base image is 58MB).

Other alternatives considered were `8-jre` and `8-jre-slim`.  Consideration was made based on the `Image Variants` section of https://hub.docker.com/_/openjdk?tab=description.

Reply:
Sweet :).